### PR TITLE
tidb-configuration-file: remove the query feedback related configuration

### PR DESCRIPTION
**Pull request by tiancaiamao**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/tidb/pull/35798 remove this configure item, update document for it.

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @lcwangchao

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@tiancaiamao Please fix the broken anchors reported by CI link check. Thanks~

**Comment by tiancaiamao** [comment]
> @tiancaiamao Please fix the broken anchors reported by CI link check. Thanks~

The lint check says 5.0/ 5.1/ 5.2 is still refering to the `/tidb-configuration-file.md#feedback-probability` url,
and if we remove this, they would become dead link.

But we only apply this on 6.2+ and master,  what's the business with 5.0/5.1/5.2

**Comment by shichun-0415** [comment]
> > @tiancaiamao Please fix the broken anchors reported by CI link check. Thanks~
> 
> The lint check says 5.0/ 5.1/ 5.2 is still refering to the `/tidb-configuration-file.md#feedback-probability` url, and if we remove this, they would become dead link.
> 
> But we only apply this on 6.2+ and master, what's the business with 5.0/5.1/5.2

It is the release notes that report dead link and release notes of all versions exist in the master branch. Therefore, if we remove an item, the link will become invalid. How about the fix in [66d1901](https://github.com/pingcap/docs-cn/pull/10752/commits/66d190194c024befbc567509c97fbf03c088bb6f)?

File: statistics.md [hunk 1]
```diff
@@ -341,7 +341,7 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 为了避免小表因为少量数据修改而频繁触发自动更新，当表的行数小于 1000 时，TiDB 不会触发对此表的自动更新。你可以通过 `SHOW STATS_META` 来查看表的行数情况。
 
-在 TiDB v5.0 之前，执行查询语句时，TiDB 会以 [`feedback-probability`](/tidb-configuration-file.md#feedback-probability) 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。**从 v5.0 起，该功能默认关闭，暂不建议开启此功能。**
+在 TiDB v5.0 之前，执行查询语句时，TiDB 会以 `feedback-probability`的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。**从 v5.0 起，该功能默认关闭，暂不建议开启此功能。从 v6.2.0 开始，该配置项废弃。**
```
Comment (TomShawn) [inline statistics.md:344]: 被删除还是被废弃呢？

**Review by TomShawn — APPROVED** [review]

File: statistics.md (hunk 1 above)
Comment (shichun-0415) [reply statistics.md:344]: 删除吧，我改下。

**Pull request merged**